src/faction_camp.cpp

```diff
@@ -2752,9 +2752,8 @@ time_duration talk_function::companion_travel_time_calc( const std::vector<tripo
     return work + one_way * trips * 1_minutes;
 }
 
-int talk_function::om_carry_weight_to_trips( units::mass mass, units::volume volume,
-        units::mass carry_mass,
-        units::volume carry_volume )
+int talk_function::om_carry_weight_to_trips( const units::mass &mass, const units::volume &volume,
+        const units::mass &carry_mass, const units::volume &carry_volume )
```

Comment:
Passing `units` types by const reference rather than value is likely going to be a pessimization.  From the ABI perspective they're effectively `int`s.

Reply:
Makes sense. I've rolled back these changes.